app/services/yelp_service.rb

```diff
@@ -0,0 +1,27 @@
+require 'yelp'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.